app/src/main/java/com/erm/artists/di/module/AppInjector.kt

```diff
@@ -0,0 +1,60 @@
+package com.erm.artists.di.module
```

Comment:
I'd move this out of `/module` and into a neww `artists/initializer` package so we can eventually utilize the hot-off-the-press [Initializer API](https://developer.android.com/topic/libraries/app-startup)